packages/api-bzz-base/src/index.ts

```diff
@@ -101,7 +107,7 @@ export async function resHex<R extends BaseResponse>(
 export async function resSwarmHash<R extends BaseResponse>(
   res: R,
 ): Promise<string> {
-  const value = await resOrError(res).arrayBuffer()
+  const value = await resOrError(res).then(res => res.arrayBuffer())
```

Comment:
Could you use `await` here please rather than mixing `await` and `.then()` code?